SUBMODULE_VERSIONS

```diff
@@ -5,7 +5,7 @@ daff5fead3fbe22c6fc58310ca3f49caf117f185 third_party/benchmark
 a5d9d0f7d368054fd1691aedf1db4116efcc233e third_party/flatbuffers
 4fb0ff7069bd88ee85902f4d0bb62794e5f6d021 third_party/flatcc
 f2fb48c3b3d79a75a88a99fba6576b25d42ec528 third_party/googletest
-93377888ae89560ba6d3976e2762d3d4724c4dfd third_party/llvm-project
+9b3c2a72e4cb3b0ae27f87064c11f728452b2af9 third_party/llvm-project
```

Comment:
@benvanik this includes https://github.com/llvm/llvm-project/commit/820e65f9e2369d2990fde4b3e7cfceb64f0df9c8